src/_actions/session.action.js

```diff
@@ -0,0 +1,14 @@
+import constants from '../_constants';
+
+const { SESSION_EXPIRED, SESSION_UNEXPIRED } = constants;
+
+/**
+ * load Asset Type thunk
+ *
+ * @return dispatch type and payload
+ */
+export const expireSession = () => ({ type: SESSION_EXPIRED });
```

Comment:
I forgot to save this comment so I'm re-writing it again 😢 

There's no need to have two different action creators for setting the boolean  value. Change `expireSession` to accept a parameter, whose value will be either `false` or `true`.  That is:
In `session.action.js`
```
export const expireSession = isSessionExpired => ({
  type: SESSION_EXPIRED,
  isSessionExpired
});
```

in `session.reducer.js`
```
export default (state = initialState.session, action) => {
  switch (action.type) {
    case SESSION_EXPIRED:
      return {
        ...state,
        sessionExpired:  action.isSessionExpired
      };
    default:
      return state;
  }
};
```

then in `SessionExpiredComponent.jsx`
```
tokenValid = () => {
  ...
  if (jwToken) {
    ...
    if (now >= tokenExpiry * 1000) {
      this.props.expireSession(true);
    }
  }
};

handleLogout = () => {
  localStorage.removeItem('art-prod-web-token');
  this.props.expireSession(false);
  this.props.history.push('/');
  };
```